src/types.rs

```diff
@@ -522,12 +522,16 @@ impl Context {
     /// Describe the Context as a human-readable string
     pub fn display(&self, sig: &Signature) -> String {
         match self {
-            Context::Hole => "<H>".to_string(),
+            Context::Hole => "[ ]".to_string(),
```

Comment:
Many languages (e.g. c) use curly braces `{ }` for lists (arrays) as well.

Since all identifiers are alphanumeric, we could have a hole be `[!]`, and still use `[]`, `[term]`, ... for pretty-printed lists. How does that sound?

Reply:
Works for me!